src/backend/web/routes/feeds.js

```diff
@@ -51,4 +51,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', async (req, res) => {
+  const feedData = req.body;
+
+  try {
+    const feed = await Feed.create(feedData);
+
+    // If the object we get back is empty, use 404
```

Comment:
This "404" block can be removed.  We'll deal with the case of the Redis call failing in your `catch` below.